app/Models/Foundation/Summit/Summit.php

```diff
@@ -2376,7 +2376,7 @@ public function getSlug($suffix = '-summit')
     public function generateRegistrationSlugPrefix(): void
     {
         if (empty($this->registration_slug_prefix)) {
-            $this->registration_slug_prefix = $this->slug;
+            $this->registration_slug_prefix = $this->getSlug();
```

Comment:
why this change is neeeded ? slug is entered ny user not generated please rollback it back

Reply:
rolled back